src/components/scanHostList/scanHostList.js

```diff
@@ -92,9 +91,9 @@ class ScanHostList extends React.Component {
 
     return (
       <div className="quipucords-infinite-results">
-        <Grid fluid onScroll={this.onScrollList} className="quipucords-infinite-list">
-          {hostsList.map(host => children({ host }))}
-        </Grid>
+        <List isPlain onScroll={this.onScrollList}>
+          {hostsList?.map(host => children({ host }))}
+        </List>
```

Comment:
required:
- looks like we might need both css classes on that div with the `onScroll` event, since one className applies width, and the other applies max-height
- and it looks like the consumers of this component don't apply `ListItem` so we'll have to apply it here too

```
        <div onScroll={this.onScrollList} className="quipucords-infinite-results quipucords-infinite-list">
          <List isPlain>
            {hostsList?.map(host => (
              <ListItem isPlain key={host.name}>
                {children({ host })}
              </ListItem>
            ))}
          </List>
        </div>
...
```


optional
- you'll probably have to leave the old css class since they're used elsewhere... create a `FixMe` note in `_view.scss` about consolidating both `quipucords-infinite-results` and `quipucords-infinite-list` into a single css class

Reply:
Tested both of these options and it looks like we can set both classes on `List` as long as we remove that outer div!